test/extensions/path/uri_template_lib/uri_template_fuzz_test.cc

```diff
@@ -0,0 +1,25 @@
+#include "source/common/common/statusor.h"
+#include "source/extensions/path/uri_template_lib/uri_template.h"
+
+#include "test/fuzz/fuzz_runner.h"
+#include "test/fuzz/utility.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  absl::string_view input(reinterpret_cast<const char*>(buf), len);
+
+  // test rewriter parser
```

Comment:
nit: according to style-guide comments begin with capital letter and end with "."

Reply:
done